packages/framework-provider-local/test/library/scheduled-adapter.test.ts

```diff
@@ -0,0 +1,44 @@
+import { BoosterConfig } from '@boostercloud/framework-types'
+import { expect } from '../expect'
+import { fake, SinonStub, stub } from 'sinon'
+import { rawScheduledInputToEnvelope } from '../../src/library/scheduled-adapter'
+import { describe } from 'mocha'
+
+describe('scheduled-adapter', () => {
+  let mockConfig: BoosterConfig
+  let loggerDebugStub: SinonStub
+
+  beforeEach(() => {
+    mockConfig = new BoosterConfig('test')
+    mockConfig.appName = 'nuke-button'
+
+    loggerDebugStub = stub()
+
+    mockConfig.logger = {
+      info: fake(),
+      warn: fake(),
+      error: fake(),
+      debug: loggerDebugStub,
+    }
+  })
+
+  describe('rawScheduledInputToEnvelope', () => {
+    ;[{ typeName: '' }, { typeName: undefined }].forEach(function (arg) {
+      it('should throw an error when typeName is empty or undefined', async () => {
+        await expect(rawScheduledInputToEnvelope(mockConfig, arg)).to.eventually.be.rejectedWith(
+          new Error(
+            `typeName is not defined or empty, scheduled command envelope should have the structure {typeName: string }, but you gave ${JSON.stringify(
+              arg
+            )}`
+          ).message
+        )
+      })
+    })
+
+    it('should return a ScheduledCommandEnvelope with the same typeName', async () => {
```

Comment:
Regarding this test file, I can't really think of additional test cases, but I can identify an improvement for this particular test case: ensuring the returned object is a `ScheduledCommandEnvelope`, i.e. that it includes a UUID.

Did you have other test cases in mind for this file, @NickSeagull?

Reply:
Nah, I think for this one it is good, no need to test schema here :)